salt/engines/http_logstash.py

```diff
@@ -6,6 +6,14 @@
 An engine that reads messages from the salt event bus and pushes
 them onto a logstash endpoint via HTTP requests.
 
+.. versionchanged:: 2017.7.3
```

Comment:
I am wondering if it is a good idea to change this behavior in a dot release. It might be better to move this to `develop`. What do you think @mirceaulinic and @cachedout?

Reply:
The way I see it, the engine has an implementation bug, so this PR does not introduce a new feature. But I agree it changes the behaviour, so I wouldn't mind moving this to develop if you think that's the most appropriate.

Reply:
I have to side with @rallytime here. We've been working hard to really limit any behavior change to develop unless there is an extenuating circumstance. I think we should move this to develop.

Reply:
No problem @cachedout @rallytime - I changed the base branch and retagged the changes.